stages/org.osbuild.greenboot.conf

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+"""
+update greenboot configuration in /etc/greenboot/greenbot.conf
+"""
+
+import sys
+from configparser import RawConfigParser
+import osbuild.api
+
+SCHEMA_2 = r"""
+"required": ["config"],
+"additionalProperties": false,
+"properties": {
+    "config": {
+      "type": "object",
+      "properties": {
+        "max_boot_attempt": {
+          "type": "int",
+          "description": "The maximum number of boot attempts before rollback",
+          "default": 3
+        },
+        "watchdog_check": {
+          "type": "boolean",
+          "description": "Validation for the presence of real time clock",
+          "default": true
+        },
+        "watchdog_grace_period": {
+          "type": "int",
+          "description": "This variable is the number of hours after an upgrade that we consider the new deployment as culprit of reboot",
+          "default": 24
+        },
+        "service_monitor": {
+          "type": "string",
+          "description": "Validates the status of the services before boot-complete.target is reached",
+          "default": "sshd NetworkManager"
+        }
+      }
+    }
+  }
+"""
+
+def main(tree, options):
+  config_file = f"{tree}/etc/greenboot/greenboot.conf"
+  config = options.get("config",{})
+  if not config:
+    return 0
+  max_boot_attempt=config.get("max_boot_attempt",3)
+  watchdog_check=config.get("watchdog_check",True)
+  watchdog_grace_period=config.get("watchdog_grace_period",24)
+  service_monitor=config.get("service_monitor","sshd NetworkManager")
+  s = "top" #section to add to conf file for configparser to work correctly
+  
+  try:
+    with open(config_file) as f:
+      file_content = "[{}]\n".format(s) + f.read() #aaded the leading section name to conf
+  except:
+    return 0
+  
+  config_object = RawConfigParser()
+  config_object.read_string(file_content)
+  config = config_object[s]
```

Comment:
You re-use the `config` name for the variable. This is quite confusing. Can't we use different names?